examples/kafka/cifar10/cifar10_kafka.ipynb

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
<p>Is there a particular reason why this helm chart over others? Edit: After using it, I can see why, pretty cool that finally someone built a cloud native operator for kafka</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/1919/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>